packages/elements/src/internals/machines/third-party/actors.ts

```diff
@@ -88,6 +88,8 @@ export const handleRedirectCallback = fromCallback<AnyEventObject, HandleRedirec
         secondFactorUrl: ClerkJSNavigationEvent.signIn,
         verifyEmailAddressUrl: ClerkJSNavigationEvent.verification,
         verifyPhoneNumberUrl: ClerkJSNavigationEvent.verification,
+        signUpUrl: ClerkJSNavigationEvent.signUp,
+        signInUrl: ClerkJSNavigationEvent.signIn,
```

Comment:
@tmilewski @LekoArts  Added this one here, because the build was breaking. I don't know if that's ok.

Reply:
Should be fine 👍